test/tests/redfish10/test_redfish10_system_reset_base.py

```diff
@@ -152,8 +145,33 @@ def workflow_tasklist_status_poller(tasklist, tasktype, timeout=180):
 
     return task_errorlist
 
+
+def get_rackhd_nodetype(nodeid):
```

Comment:
it seems that many files have the same function, and the details are similar but not the same completely
```
$ grep -rn get_rackhd_nodetype * | grep def
test/tests/switch/test_rackhd20_switch_telemetry.py:43:def get_rackhd_nodetype(nodeid):
test/tests/switch/test_rackhd20_switch_pollers.py:42:def get_rackhd_nodetype(nodeid):
test/tests/switch/test_rackhd11_switch_pollers.py:42:def get_rackhd_nodetype(nodeid):
test/tests/switch/test_rackhd11_switch_telemetry.py:43:def get_rackhd_nodetype(nodeid):
test/tests/compute/test_rackhd20_computenode_pollers.py:26:def get_rackhd_nodetype(nodeid):
test/tests/compute/test_rackhd11_computenode_pollers.py:26:def get_rackhd_nodetype(nodeid):
test/common/test_api_utils.py:49:def get_rackhd_nodetype(nodeid):
```